sycl/include/CL/sycl/access/access.hpp

```diff
@@ -44,7 +44,9 @@ enum class address_space : int {
   private_space = 0,
   global_space,
   constant_space,
-  local_space
+  local_space,
+  device_space = 11,
```

Comment:
Any reason to set specific value here?

Reply:
No, there is no need to set them. Will unset.